virttest/qemu_devices/qcontainer.py

```diff
@@ -161,6 +161,7 @@ def get_qmp_cmds(qemu_binary, workaround_qemu_qmp_crash=False):
         # escape the '?' otherwise it will fail if we have a single-char
         # filename in cwd
         self.__device_help = self.execute_qemu("-device \? 2>&1", 10)
+        self.__object_help = self.execute_qemu("-object \? 2>&1", 10)
```

Comment:
in old qemu, -object \? is not supported, if it's not supported, we need to return a ""